docs/getting-started/coverage.md

```diff
@@ -0,0 +1,53 @@
+# Scanning Coverage
+
+## Programming languages
+Trivy detects code dependencies for the purpose of SBOM generation and vulnerability detection. The following programming languages have some level of support:
+
+- Ruby
+- Python
+- PHP
+- Node.js
+- .NET
+- Java
+- Go
+- Rust
+- C/C++
+- Elixir
+- Dart
+- Swift
+
+Some features might be not be universally supported. For a full supportability matrix, [see here](../docs/vulnerability/detection/language.md).
+
+## Package managers
```

Comment:
Technically, Trivy supports three package managers, apt, yum (dnf) and apk. OS is more accurate here.

```suggestion
## OS
```